src/drivers/pwm_esc/pwm_esc.cpp

```diff
@@ -432,6 +443,14 @@ PWMESC::task_main()
 
 	PX4_DEBUG("exiting");
 
+	/* Configure PWM to default rate, 0 pulse and stop */
+
+	int n_pwm_devices = 1;
+
+	for (int i = 0; i < 1 && i < n_pwm_devices; i++) {
+		set_defaults(_pwm_fd[i], PWMIOC_STOP);
+	}
+
 	/* tell the dtor that we are exiting */
 	_task = -1;
 	_exit(0);
```

Comment:
If this is the kernel process that is spawned and runs until stopped, I don't think calling _exit() here is necessary. The common kernel side exit logic should handle this just fine.

Reply:
That's right, it seems that I just missed that. I will remove the _exit here...